MapboxNavigation/PollyVoiceController.swift

```diff
@@ -46,6 +46,8 @@ public class PollyVoiceController: RouteVoiceController {
     
     var spokenInstructionsForRoute = NSCache<NSString, NSData>()
     
+    var previousInstruction: String?
```

Comment:
Rename this property to `lastSpokenInstruction` and set it to an Instruction, not just its SSML text.